lib/mayaUsd/commands/layerEditorCommand.cpp

```diff
@@ -278,8 +280,44 @@ class InsertRemoveSubPathBase : public BaseCmd
         }
     }
 
+    void saveSelection()
```

Comment:
I was hoping to have another pass as a refactor task and go through all commands in that file and assess which ones need the selection store / restore. I have seen similar code in AL USD so this might be good to go into a global location that can be used from various UI.

Reply:
@seando-adsk I made a follow up ticket for this request MAYA-129060 . Thank you.